.github/workflows/build-evmos.yml

```diff
@@ -0,0 +1,81 @@
+name: docker-build
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
+  pull_request:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: celestiaorg/evmos
+  TAG_PREFIX: optimint-
+
+jobs:
+  build:
+    runs-on: "ubuntu-latest"
+    permissions:
+      contents: write 
+      packages: write 
+    steps:
+      - name: "Checkout source code"
+        uses: "actions/checkout@v2"
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: up a level
+        run: cd .. && pwd
+      - name: Clone evmos repo
+        uses: actions/checkout@v2
+        with:
+          repository: jbowen93/evmos
+          path: evmos
```

Comment:
Should this repo to moved to celestiaorg soon?

Reply:
Yes this will be done when I merge stuff back to celestiaorg https://github.com/celestiaorg/evmos/pull/32